<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormEditarProduto</class>
 <widget class="QDialog" name="FormEditarProduto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLineEdit, QSpinBox, QDoubleSpinBox{
	border: 1px solid #bfbebd;
	border-radius: 10px;
	padding: 2px;
}

QSpinBox::up-button, QDoubleSpinBox::up-button  {
	subcontrol-origin: margin;
	subcontrol-position: top right;
	right: 5px;
}
QSpinBox::down-button, QDoubleSpinBox::down-button {
	subcontrol-origin: border;
	subcontrol-position: bottom right;
	right: 5px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <item>
    <widget class="QLabel" name="label_7">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&lt;div style=&quot;text-align: center;&quot;&gt;&lt;h1&gt;Editar Produto&lt;/h1&gt;&lt;/div&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Código do produto:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="codProd">
       <property name="styleSheet">
        <string notr="true">color: #bfbebd;</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Descrição do produto:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="descricao"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Quantidade:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="quantidade">
       <property name="maximum">
        <number>9999</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Valor de compra:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="valorCompra">
       <property name="prefix">
        <string>R$ </string>
       </property>
       <property name="maximum">
        <double>999999.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Valor de venda:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="valorVenda">
       <property name="prefix">
        <string>R$ </string>
       </property>
       <property name="maximum">
        <double>999999.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="confirmarButton">
     <property name="styleSheet">
      <string notr="true">QPushButton {
	margin-top: 10px;
	margin-bottom: 5px;
	padding: 5px;
	border-radius: 10px;
	background-color: #26a844;
	color: #fff;
}

QPushButton:hover {
	background-color: #22973d;
}</string>
     </property>
     <property name="text">
      <string>Confirmar alterações</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="cancelarButton">
     <property name="styleSheet">
      <string notr="true">QPushButton {
	margin-top: 5px;
	margin-bottom: 5px;
	padding: 4px;
	border: 1px solid #dc3643;
	border-radius: 10px;
	background-color: #fff;
	color:#dc3643;
}

:hover {
	background-color: #dc3643;
	color: #fff;
}</string>
     </property>
     <property name="text">
      <string>Cancelar</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
